Library/Formula/testssl.rb

```diff
@@ -0,0 +1,13 @@
+class Testssl < Formula
+  desc "Tool which checks for the support of TLS/SSL ciphers and flaws"
+  homepage "https://testssl.sh/"
+  url "https://github.com/drwetter/testssl.sh/archive/v2.6.tar.gz"
+  version "2.6"
+  sha256 "286b3285f096a5d249de1507eee88b14848514696bc5bbc4faceffa46b563ebd"
+
+  depends_on "openssl"
+
+  def install
+    bin.install "testssl.sh"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!